src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -19,7 +19,16 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="Microsoft.DotNet.Build.CloudTestTasks\*.dll" target="lib\net45" />
+    <file src="Microsoft.DotNet.Build.Tasks.net45\*.dll" target="lib\net45" />
+    <file src="Microsoft.DotNet.Build.CloudTestTasks\Microsoft.Azure.KeyVault.Core.dll" target="lib\net45" />
```

Comment:
Why did you need to expand this instead of using *.dll now? Is it because there are file conflicts?